src/cmd/vm.rs

```diff
@@ -20,136 +27,231 @@ pub struct Args {
 #[derive(FromArgs, PartialEq, Debug)]
 #[argh(subcommand)]
 enum SubCommand {
-    Connect(ArgsConnect),
     Setup(ArgsSetup),
     Start(ArgsStart),
-    Push(ArgsPush),
 }
 
 #[tracing::instrument(level = "trace")]
 pub fn run(args: &Args) -> Result<()> {
     let config = Config::read()?;
     if !config.is_internal() {
-        return Err(anyhow!(
-            "vm subcommand is currently only supported for google internal use"
-        ));
+        bail!("vm subcommand is currently only supported for google internal use");
     }
 
     match &args.nested {
-        SubCommand::Connect(args) => run_connect(args),
         SubCommand::Setup(args) => run_setup(args),
         SubCommand::Start(args) => run_start(args),
-        SubCommand::Push(args) => run_push(args),
     }
 }
 
 #[derive(FromArgs, PartialEq, Debug)]
-/// connects to a running betty instance via SSH
-#[argh(subcommand, name = "connect")]
-pub struct ArgsConnect {
-    /// extra arguments
+/// run first time setup, installs necessary dependencies
+#[argh(subcommand, name = "setup")]
+pub struct ArgsSetup {
+    /// path to dir where betty.sh exists. If omitted, current directory will be
+    /// used.
+    #[argh(option)]
+    arc: Option<String>,
+
+    /// extra arguments to pass to betty.sh. You can pass other options like
+    /// --extra-args "options".
     #[argh(option)]
-    extra_args: Vec<String>,
+    extra_args: Option<String>,
 }
 
-fn run_connect(_args: &ArgsConnect) -> Result<()> {
+fn run_setup(args: &ArgsSetup) -> Result<()> {
+    let dir = find_betty_script(&args.arc)?;
+
+    println!("Updating packages...");
+    let update_package = Command::new("sudo")
+        .args(["apt", "update"])
+        .spawn()
+        .context("Failed to execute sudo apt update")?;
+    update_package
+        .wait_with_output()
```

Comment:
throughout this file you use "wait_with_output" but don't use the output.

I think this should be changed to "wait"